src/components/EditProfile/index.jsx

```diff
@@ -78,38 +78,104 @@ const EditProfile = ({ userData, username, setIsEditing, setUserData }) => {
     }
   };
 
-  const handleImgSave = () => {
-    if (!usernameAvailable) {
-      return;
+  const [error, setError] = useState({});
+  const handleImgSave = (event) => {
+    event.preventDefault();
+    console.log("called");
+    console.log(editedData.name);
```

Comment:
remove console logs as they're not required in production.... (though maybe I should add a plugin later to automatically remove them)